### PR TITLE
New front end tests

### PR DESCRIPTION
**Pull request by chwawa**
# Pull Request Summary

## Description

This PR is for a few frontend tests.

## Type of change

Please delete options that aren't relevant.

- [x] Creating tests

## Any questions, comments, concerns, or next steps? 
A few tests aren't working because navigation isn't happening after a button click. I'll look into this.

### CODE REVIEW

File: front-end/package.json [hunk 1]
```diff
@@ -12,30 +14,35 @@
...
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.7",
     "history": "^5.3.0",
+    "jest-environment-jsdom": "^29.3.1",
     "lodash": "^4.17.21",
     "node": "^16.0.0",
     "numeral": "^2.0.6",
     "prop-types": "^15.8.1",
-    "react": "^16.14.0",
+    "react": "^18.2.0",
```
Comment (ProEdge2) [inline front-end/package.json:35]: Was there a reason you upgraded the required versions of react? and the Dom?

File: front-end/src/__test__/AddProject.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/**
+ * @jest-environment jsdom
+ */
```
Comment (ProEdge2) [inline front-end/src/__test__/AddProject.test.js:3]: I like the testing

**Review by ProEdge2 — COMMENTED** [review]
tests look great Chelsea!

File: front-end/package.json (hunk 1 above)
Comment (chwawa) [reply front-end/package.json:35]: Yes--I had some trouble getting the testing environment set up at first. Researching on Stack Overflow and upgrading some stuff (like React, react-test-renderer, etc.) helped.

**Review by ProEdge2 — APPROVED** [review]

**Pull request merged**